lang/polyml/Portfile

```diff
@@ -1,27 +1,25 @@
 PortSystem 1.0
+PortGroup           github 1.0
 
-name                polyml
-version             5.5.2
+github.setup        polyml polyml 5.7.1 v
 categories          lang ml
 platforms           darwin
-maintainers         nomaintainer
+maintainers         {veonix.com:phil.clayton @pclayton}
```

Comment:
Can open maintainer be added here?

Reply:
Ditto. Unless there's a good reason not to have it openmaintainer, make it openmaintainer.

Reply:
Yes, I'll add it.

Reply:
Now added.